overlord/managers_test.go

```diff
@@ -160,6 +152,41 @@ func (ms *mgrsSuite) SetUpTest(c *C) {
 	err = os.MkdirAll(filepath.Dir(snapSeccompPath), 0755)
 	c.Assert(err, IsNil)
 	ms.snapSeccomp = testutil.MockCommand(c, snapSeccompPath, "")
+
+	o, err := overlord.New()
+	c.Assert(err, IsNil)
+	ms.o = o
+	st := ms.o.State()
+	st.Lock()
+	defer st.Unlock()
+	st.Set("seeded", true)
+
+	// add "core" snap declaration
+	headers := map[string]interface{}{
+		"series":       "16",
+		"snap-name":    "core",
+		"publisher-id": "can0nical",
+		"timestamp":    time.Now().Format(time.RFC3339),
+	}
+	headers["snap-id"] = fakeSnapID(headers["snap-name"].(string))
+	a, err := ms.storeSigning.RootSigning.Sign(asserts.SnapDeclarationType, headers, nil, "")
```

Comment:
Maybe I'm missing something, but when I just use the "storeSigning" I get an error like this:
```
FAIL: managers_test.go:105: mgrsSuite.SetUpTest

managers_test.go:174:
    c.Assert(err, IsNil)
... value *errors.errorString = &errors.errorString{s:"no matching public key \"XcSmSI47H8EdldbP1BMKBUXs_5Jr0Ck2wSecjcExc5KMEAzsfZocbpaoSvQAW7_9\" for signature by \"can0nical\""} ("no matching public key \"XcSmSI47H8EdldbP1BMKBUXs_5Jr0Ck2wSecjcExc5KMEAzsfZocbpaoSvQAW7_9\" for signature by \"can0nical\"")
```